include/revng/BasicAnalyses/AdvancedValueInfo.h

```diff
@@ -102,6 +109,12 @@ getUniqueUnknown(llvm::ScalarEvolution &SE, const llvm::SCEV *SC) {
   return FSU.UniqueUnknown;
 }
 
+namespace detail {
+using Graph = llvm::DenseMap<
+  llvm::BasicBlock *,
+  std::unique_ptr<llvm::SmallVector<llvm::BasicBlock *, 2>>>;
+}
```

Comment:
Can we have a `using GraphValue = std::unique_ptr<llvm::SmallVector<llvm::BasicBlock *, 2>>`?

Reply:
not sure what this would imply

Reply:
We have this type twice in the codebase and it's rather obscure.
Plus: `using Graph = llvm::DenseMap<llvm::BasicBlock *, GraphValue>` fits in a single line!